### DISCUSSION TITLE
Lots of errors in Logalerts since this morning for static files

### DISCUSSION THREAD

**Discussion by lionelrudaz** [vercel/community#226] [Help] [answered]
Hey folks,

I've reported the issue this morning on Twitter: https://twitter.com/lionelrudaz/status/1485914746570387461

Since this morning, I start receiving a lot of alerts by Logalerts for static files. Here's a screenshot:

<img width="655" alt="CleanShot 2022-01-25 at 13 40 11@2x" src="https://user-images.githubusercontent.com/592678/150978630-918a3020-163c-418f-b6f2-34c3d2a7b8a2.png">

I've not deployed to production for a few days, so I was wondering if there was an issue with the platform, or if it was Logalert that's going bonkers. I don't see anything on the Chrome console, nothing in the Function logs in Vercel or in Logtail.

My website is https://www.tasters.ch

Anybody running on the same issue?

Cheers!

**Comment by JTaylor0196** [comment, marked as answer]
Hey there, thanks for your input! As a result of this feedback, the team is looking into this and we'll let you know what we find. Stay tuned!

**Reply by lionelrudaz** [reply]
Thanks @JTaylor0196, for the moment, I still encounter the issue.

I'm planning to submit new code today or tomorrow. Perhaps this will have an effect.

Cheers!

**Reply by JTaylor0196** [reply]
Hey there @lionelrudaz! Circling back to update you and let you know that the team created and rolled out a fix for this issue today. Please let me know if you still happen to experience this issue, but, moving forward, you should be all set.

Thanks for your patience while we've worked through this!

**Reply by lionelrudaz** [reply]
Thanks @JTaylor0196 and the team, nothing showing up for the last couple days. 

Cheers!
